py/nupic/examples/mnist.py

```diff
@@ -140,8 +130,8 @@ def main(parameters=default_parameters, argv=None, verbose=True):
     # Training Loop
     for i in range(len(train_images)):
         img, lbl = random.choice(training_data)
-        enc.encode(np.squeeze(img))
-        sp.compute( enc.output.flatten(), True, columns )
+        enc.dense = img >= np.mean(img)
```

Comment:
please add a comment that this "img >= mean(img)" converts to black/white (am I right?). 
Why comparing to mean, and not 0.5 if the color is greyscale?

Reply:
I use the mean here because it what was used before this PR. I don't want to change too much, just to fix a bug and clean up a little.

Reply:
ok, after the PR we can test with 0.5 instead and compare the score